docker_dev.sh

```diff
@@ -3,7 +3,7 @@ docker build -f alpine.docker -t diginc/pi-hole:alpine .
 docker tag diginc/pi-hole:alpine diginc/pi-hole:latest
 docker build -f debian.docker -t diginc/pi-hole:debian .
 
-IP=$(ip addr show eth0 | grep "inet\b" | awk '{print $2}' | cut -d/ -f1)
+IP=$(ip route get 8.8.8.8 | awk '{ print $NF; exit }')
```

Comment:
Knowing about the openVPN corner case I'm tempted to deny the change.  I know we could add more logic to say use this when the interface != tun0 but keep in mind I duplicate this command in the README (line 20) too, so I want it to be as short and concise as possible. 

Right now users who don't have eth0 probably know they don't have eth0 and just change NIC to en0/wlan0 or whatever.  I don't get a lot of feedback on the startup steps not working so it is partially a case of: if it isn't broke don't fix it too.